src/main/java/com/avispl/symphony/dal/device/core510i/QSYSCoreCommunicator.java

```diff
@@ -130,6 +249,54 @@ protected LoginInfo initLoginInfo() {
 		return new LoginInfo();
 	}
 
+	/**
+	 * Init instance of QRCCommunicator
+	 *
+	 * @throws Exception if init fail
+	 */
+	public void initQRCCommunicator() throws Exception {
+		qrcCommunicator = new QRCCommunicator();
+		qrcCommunicator.setHost(this.host);
+		qrcCommunicator.setPort(QSYSCoreConstant.QRC_PORT);
```

Comment:
will it make sense to make qrc setting (port in this case) configurable through adapter properties, with default value, if it isn't set?

Reply:
Qsys use Rest as the main communicator, and for QRC, the port is always 1710, so I think we don't need to make it configurable through adapter properties.